app/src/main/java/se/sigmaconnectivity/blescanner/ui/home/HomeViewModel.kt

```diff
@@ -35,4 +39,11 @@ class HomeViewModel(private val contactUseCase: ContactUseCase) : BaseViewModel(
     fun turnOffLEService() {
         mutableLEServiceStatus.value = FeatureStatus.INACTIVE
     }
+
+    fun setPhoneNumberHash(hash: String) {
```

Comment:
What's the difference if it's from phone number or something else?
```suggestion
    fun setPHash(hash: String) {
```

Reply:
maybe.. on the other hand what if we would like to add later another hash (e.g. username or ID)? also we use phone in JS bridge aligned with web team so maybe better to keep it consistent?